string object_name
---
bool success
string arm
int32 grasp_id
int32 num_grasps_tried

